src/vstest.console/Resources/Resources.Designer.cs

```diff
@@ -39,7 +39,7 @@ internal class Resources {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("vstest.console.Resources.Resources", typeof(Resources).Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.VisualStudio.TestPlatform.CommandLine.Resources.Resources", typeof(Resources).Assembly);
```

Comment:
Is this fixing the resources regeneration? I know that I have problems with the correct namespace in vstest.console, but I am not sure now which way it goes. So this either fixes it or breaks it. I guess this is a fix?

Reply:
Saw the root namespace change below, disregard this.